zkevm-circuits/src/bytecode_circuit/circuit.rs

```diff
@@ -613,6 +620,9 @@ impl<F: Field> BytecodeCircuitConfig<F> {
         length: F,
         push_data_size: F,
     ) -> Result<(), Error> {
+        // annotate columns
+        self.annotate_circuit(region);
```

Comment:
If this function is here, it will be called many times (one for each row), but we only need to call it once per region.  So I suggest moving this call to here: https://github.com/privacy-scaling-explorations/zkevm-circuits/blob/6396dfd8d214fc22fe3978aec70c00bd38fcd119/zkevm-circuits/src/bytecode_circuit/circuit.rs#L445
where the region is created.

Reply:
fixed in [[99cd564](https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/1152/commits/99cd564d5db77197f476bbb7d82a415823ad6572)